src/content/docs/apm/agents/go-agent/configuration/distributed-tracing-go-agent.mdx

```diff
@@ -64,8 +57,9 @@ To start the installation routine, click the tile below. When you are finished i
     icon={<img variant="TechTile" src={apmGoLogo} alt="Golang"/>}
```

Comment:
@nbaenam I think it's OK to have "Start installation" because it's in the context of the particular agent, there's an associated image, and there's an alt-text description.